static/scss/detail/tiles.scss

```diff
@@ -0,0 +1,95 @@
+// sass-lint:disable mixins-before-declarations
+
+.tiles-block {
+  width: 100%;
+  padding: 32px 0;
```

Comment:
It looks like this tiles section has left and right padding of 15px because of some default style definition for `.container`. I actually can't tell where that is being defined. In general the definitions for the tile spacing are all over the place. Can you try making the following changes?
- Set `padding: 0` for `.container` at the top of this file
- In `.tiles-block`, change the padding definition to `padding: 32px 7px;`
- In `.tiles-list`, remove `margin: 0 -8px;`

Reply:
addressed all other suggestion but that change is effecting UI. Can we go with the existing one ?

Reply:
@asadiqbal08 sorry, this should do it:
- Set `padding: 0` for `.tiles-block .container`
- In `.tiles-list`, set `margin: 0;` and `padding: 0 7px;`